src/spec/spec.cr

```diff
@@ -209,7 +209,7 @@ OptionParser.parse! do |opts|
     exit
   end
   opts.on("-v", "--verbose", "verbose output") do
-    Spec.formatter = Spec::VerboseFormatter.new
+    Spec.formatters << Spec::VerboseFormatter.new
```

Comment:
You must do `[ ... ] of Spec::Formatter`, otherwise the compiler will deduce it as an `Array(Spec::VerboseFormatter)`, and you can't insert any formatter into that array.

Instead of replacing with `Spec.formatters = [...]`, which forces you to specify that `of Spec::Formatter`, a simpler approach is to just to:

```crystal
Spec.formatters.clear
Spec.formatters << Spec::VerboseFormatter.new
```

Or just:

```crystal
Spec.formatters.replace [Spec::VerboseFormatter.new]
```

Here `Array#replace` just copies the elements into the existing array, so it'll work.